File: apps/benchmarks/micro/01_counter.py
#!/usr/bin/env python3
# ベンチマーク1: シンプルカウンター (Python版)
# 期待値: 10

def main():
    i = 0
    while i < 10:
        i = i + 1
    print(f"Result: {i}")
    return 0

if __name__ == "__main__":
    main()

File: apps/benchmarks/micro/01_counter.c
// ベンチマーク1: シンプルカウンター (C言語版)
// 期待値: 10

#include <stdio.h>

int main() {
    int i = 0;
    while (i < 10) {
        i = i + 1;
    }
    printf("Result: %d\n", i);
    return 0;
}
